From alan@linux.intel.com  Tue Jul 27 10:12:52 2010
From: Alan Cox <alan@linux.intel.com>
Subject: serial: max3107: Fix gpiolib support
To: linux-kernel@vger.kernel.org, linux-serial@vger.kernel.org, greg@kroah.com
Date: Tue, 27 Jul 2010 08:16:05 +0100
Message-ID: <20100727071551.25761.146.stgit@localhost.localdomain>

Because of the way gpiolib works we actually need to ifdef this in our
header file

Signed-off-by: Alan Cox <alan@linux.intel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/serial/max3107.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/serial/max3107.h
+++ b/drivers/serial/max3107.h
@@ -368,8 +368,10 @@ struct max3107_port {
 	/* SPI device structure */
 	struct spi_device *spi;
 
+#if defined(CONFIG_GPIOLIB)
 	/* GPIO chip stucture */
 	struct gpio_chip chip;
+#endif
 
 	/* Workqueue that does all the magic */
 	struct workqueue_struct *workqueue;
